node/service/src/chain_spec/moonbase.rs

```diff
@@ -269,7 +269,9 @@ pub fn testnet_genesis(
 		},
 		proxy_genesis_companion: Default::default(),
 		treasury: Default::default(),
-		migrations: Default::default(),
+		migrations: MigrationsConfig {
+			completed_migrations: runtime_common::migrations::migrations_names::<Runtime>(),
```

Comment:
I always like the name `prehistoric_migrations` for this field because it really emphasizes that these migrations were "completed" at some point before we even started keeping track. That differentiates them from migrations that were completed earlier in the chain history.